pkgs/development/libraries/libgda/default.nix

```diff
@@ -65,6 +61,10 @@ stdenv.mkDerivation rec {
     postgresql
   ];
 
+  propagatedBuildInputs = [
+    libxml2
```

Comment:
`lib/pkgconfig/libgda-5.0.pc`: `Requires: gobject-2.0 gthread-2.0 libxml-2.0`

Reply:
gthread is glib? and should we move gobject from native to propagated?